### PR TITLE
Bug 1826091 - Enable Pull to Refresh in release

### PR DESCRIPTION
**Pull request by onemahon**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
  * I couldn't find any apparent automated tests for checking presence of features in different channels
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1826091

### CODE REVIEW

**Review by rahulsainani — APPROVED** [review]
✅ You might want to squash the commits into 1, that would also fix the the bugzilla commit format issue as the first commit follows the format.

**Comment by onemahon** [comment]
Ah, bummer, I was planning on using the "squash and merge" button to do that, but it seems that the format error disables it until I locally rebase into a single commit. 

Thanks for the quick look!

File: docs/changelog.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ permalink: /changelog/
 * **concept-engine**:
   * Add new `hasCookieBannerRuleForSession` API in `Engine`. This is currently only supported in the Gecko Engine.
 
+* **pull-to-refresh**:
+  * Enable 'pull to refresh' feature in all channels.
+
```
Comment (jonalmeida) [inline docs/changelog.md:51]: Non-blocking comment: the changelog title is for the component we add a feature towards.

You could omit this entry.

File: docs/changelog.md [hunk 2]
```diff
@@ -46,6 +46,9 @@ permalink: /changelog/
 * **concept-engine**:
   * Add new `hasCookieBannerRuleForSession` API in `Engine`. This is currently only supported in the Gecko Engine.
 
+* **pull-to-refresh**:
+  * Enable 'pull to refresh' feature in all channels.
```
Comment (kycn) [inline docs/changelog.md:50]: nit: is the intended form `pull to refresh`?  (with ` instead of ')

File: docs/changelog.md (hunk 1 above)
Comment (jonalmeida) [reply docs/changelog.md]: We don't have a process around this yet, hence the recommendation to omit it for now.

We probably want to do something like this in the future:

```markdown
* **fenix**
  *  Enable 'pull to refresh' feature in all channels.

* **focus**
  * Enable 'pull to refresh' feature in nightly-only.
```

**Comment by jonalmeida** [comment]
@Mergifyio backport releases_v112

**Pull request merged**